pixeltable/tests/conftest.py

```diff
@@ -38,10 +38,40 @@ def init_env(tmp_path_factory) -> None:
 
 @pytest.fixture(scope='function')
 def test_client(init_env) -> pt.Client:
+    # Clean the DB *before* instantiating a client object. This is because some tests
+    # (such as test_migration.py) may leave the DB in a broken state, from which the
+    # client is uninstantiable.
+    clean_db()
     cl = pt.Client(reload=True)
     cl.logging(level=logging.DEBUG, to_stdout=True)
     yield cl
-    cl.reset_catalog()
+
+
+def clean_db(restore_tables=True):
+    from pixeltable.env import Env
+    # (aaron-siegel) The logic from Client.reset_catalog() has been moved here, so that it
```

Comment:
Why include your name?

Reply:
I've tried to get in the habit of including my name on TODOs. But you're right, this isn't a TODO. And git blame can be used if it's really needed. I'll remove it